test/e2e/bundler.ts

```diff
@@ -0,0 +1,66 @@
+import { startBundler, BundlerServer } from '@metamask/test-bundler';
+import { BUNDLER_URL } from './constants';
+
+const CONFIG_FILE = `${__dirname}/bundler.config.json`;
+
+export class Bundler {
+  #server: BundlerServer | undefined;
+
+  #userOperationHashes: string[] = [];
+
+  async start() {
+    console.log('Starting bundler');
+
+    try {
+      this.#server = await startBundler({
+        configFile: CONFIG_FILE,
+        unsafe: true,
+      });
+
+      this.#server.hub.on('user-operation-added', (userOperationHash) => {
+        this.#userOperationHashes.push(userOperationHash);
+      });
+
+      await this.#server.asyncStart();
```

Comment:
I think `startBundler` already [calls `asyncStart` via `runBundler`](https://github.com/MetaMask/test-bundler/blob/main/src/bundler/runBundler.ts#L250):

Reply:
It does, but `asyncStart` is just a reference to a promise variable so we use it to wait for the server to finish it's initialisation checks and be considered ready.